libs/core/src/components/sage-button/sage-button.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-button',
+  styleUrl: 'sage-button.scss',
+  shadow: true,
+})
+export class SageButton {
+
+  // alignEnd: false,
+  // className: '',
+  // customContentClassName: null,
+  // disclosure: false,
+  // fullWidth: false,
+  // hasCustomContent: false,
+  // icon: null,
+  // iconOnly: false,
+  // iconPosition: Button.ICON_POSITIONS.LEFT,
+  // loading: false,
+
+
+  /**
+   * Button type
+   * @defaultValue primary
+   */
+  @Prop() variant: 'accent' | 'secondary' | 'disclosure' | 'destructive' | 'primary' = 'primary';
+  /* TODO: Figure out why primary isn't setting as the default */
+
+  /**
+   * When set to true, icon is added to button
+   * @defaultValue false
+   */
+  @Prop() icon: string;
+  
+  /**
+   * When set to true, buttons are disabled
+   * @defaultValue false
+   */
+  @Prop({ reflect: true }) disabled: true;
```

Comment:
We had a convo about `reflect` in Slack. I send you the thread.

Reply:
Sounds like we likely don't need this right now.